hack/bump-kwok.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2021 Red Hat, Inc.
```

Comment:
Could you update the copyright here to "The KubeVirt Authors"

Similar to how it is here - https://github.com/kubevirt/kubevirt/blob/05904ce9261eb763238109b725cad207ad4fff72/pkg/virt-config/deprecation/passt.go#L16